task4/iot/apps/device/mix.exs

```diff
@@ -19,7 +19,8 @@ defmodule Device.Mixfile do
   # Type "mix help compile.app" for more information
   def application do
     [applications: [:logger],
```

Comment:
We're missing `:dnssd` in the list of applications. `iex -S mix` doesn't start without it.